### DISCUSSION TITLE
How do i get the ff.cfg

### DISCUSSION THREAD

**Discussion by Linusisnice** [keirf/flashfloppy#841] [Q&A]
I cant find where i can get the ff.cfg. I have a gotec drive with ff on it but there is no ff.cfg on my usb stick. im trying to use it with an old compaq pc.

**Comment by tenuke** [comment]
"Download" https://github.com/keirf/flashfloppy

pe 3. marrask. 2023 klo 15.44 Linusisnice ***@***.***>
kirjoitti:

**Reply by Linusisnice** [reply]
where is the file

**Comment by Linusisnice** [comment]
it makes an image_a.cfg but no ff.cfg

**Comment by tenuke** [comment]
Download the zip from the download page and find example ff.cfg from the
package.
Or just create your own ff.cfg with like notepad or such

pe 3. marrask. 2023 klo 16.54 Linusisnice ***@***.***>
kirjoitti:
